src/spring/azext_spring/commands.py

```diff
@@ -241,6 +253,23 @@ def load_command_table(self, _):
         g.custom_command('bind', 'service_registry_bind')
         g.custom_command('unbind', 'service_registry_unbind')
 
+    with self.command_group('spring dev-tool', is_preview=True,
+                            custom_command_type=dev_tool_portal_cmd_group,
+                            exception_handler=handle_asc_exception) as g:
+        g.custom_show_command('show', 'show',
+                              table_transformer=transform_dev_tool_portal_output)
+        g.custom_command('create', 'create', table_transformer=transform_dev_tool_portal_output, validator=validate_dev_tool_portal, supports_no_wait=True)
+        g.custom_command('update', 'update', table_transformer=transform_dev_tool_portal_output, validator=validate_dev_tool_portal, supports_no_wait=True)
+        g.custom_command('delete', 'delete')
```

Comment:
Do we need consider adding `confirmation=true` for the `delete` command?

Reply:
Delete this component is not a sensetive operation, and we want to keep consistent with all other component. So I don't add confirmation here.

Reply:
OK, got it`